components/InfoSectionCompanyEvents/index.tsx

```diff
@@ -0,0 +1,53 @@
+import styles from "./styles.module.css";
+import InfoSectionWrapper from "@/components/InfoSectionWrapper";
+import Link from "@/components/Link";
+
+const InfoSectionCompanyEvents = () => {
+  return (
+    <InfoSectionWrapper
+      id="labamba"
```

Comment:
that's not strictly speaking completely true right

Reply:
Oops, seems I have gotten myself busted in shamelessly copying the labamba section 😓